_translator-files/po/de/Running-a-Server.po

```diff
@@ -334,72 +351,81 @@ msgid ""
+"|**Genre Jazz** |`jazz.jamulus.io:22324`|\n"
+"|**Genre Classical/Folk** |`classical.jamulus.io:22524`|\n"
+"|**Genre Choral/Barbershop** |`choral.jamulus.io:22724`|\n"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:129
 msgid "You can also specify a [Directory](#3-directory) in the same way from the command line, providing the Server Address in the same format."
-msgstr ""
+msgstr "Du kannst auch ein [Verzeichnis](#server-typen) auf dieselbe Weise in der Befehlszeile angeben, indem du die Serveradresse im selben Format angibst."
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:131
 msgid "#### Running as a Directory"
-msgstr ""
+msgstr "#### Als Verzeichnis Server betreiben"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:133
 msgid "If you wish to run a [Directory](Running-a-Server#3-directory) please see [this guide](Custom-Directories)."
-msgstr ""
+msgstr "Wenn du einen [Verzeichnis Server](Running-a-Server#3. Verzeichnis Server) betreiben möchtest, lese bitte [diese Anleitung](Custom-Directories)."
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:135
 msgid "### Maintenance"
-msgstr ""
+msgstr "### Wartung"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:137
 msgid "#### Viewing The Logs"
-msgstr ""
+msgstr "#### Anzeigen der Protokolle - Log Files"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:139
 msgid "Jamulus will log to the system log file if you left the `StandardOutput=journal` setting in the unit file."
-msgstr ""
+msgstr "Jamulus protokolliert in der Systemprotokolldatei, wenn du die Einstellung `StandardOutput=journal` in der Unit-Datei nicht verändert hast."
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:141
 msgid "To view the log, use `journalctl` (to exit press Ctrl-C). For example, to read the system log file, filtered for the Jamulus service:"
-msgstr ""
+msgstr "Um das Protokoll einzusehen, verwende `journalctl` (zum Beenden drücke Ctrl-C). Um zum Beispiel die Systemprotokolldatei zu lesen, benutze den Filter nach dem Jamulus-Dienst:"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:143
 msgid "`journalctl -f -u jamulus-headless`"
-msgstr ""
+msgstr "`journalctl -f -u jamulus-headless`"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:145
 msgid "#### Controlling Recording"
-msgstr ""
+msgstr "#### Steuerung der Aufzeichnung"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:147
 msgid "When using the recording function with the `-R` command line option, if the Server receives a SIGUSR1 signal during a recording, it will start a new recording in a new Directory. SIGUSR2 will toggle recording enabled on/off."
-msgstr ""
+msgstr "Wenn die Aufzeichnungsfunktion mit der Befehlszeilenoption \"R\" verwendet wird und der Server während einer Aufzeichnung ein SIGUSR1-Signal empfängt, beginnt er eine neue Aufzeichnung in einem neuen Verzeichnis. SIGUSR2 schaltet die Aufzeichnungsfunktion ein und aus."
```

Comment:
As in multiple other places, I think this should be a back-tick: \` not a normal "